src/lexicon/pytrie_gen.cpp

```diff
@@ -100,7 +100,11 @@ getPureGBEncoding(const char* utf8str)
     static iconv_t ric_gbk = iconv_open("UTF-8", "GBK");
 
     // FIXME
-    if (ic_gb == -1 || ic_gbk == -1 || ric_gb == -1 || ric_gbk == -1) return 3;
+    if (ic_gb   == (iconv_t)-1 ||
+        ic_gbk  == (iconv_t)-1 ||
+        ric_gb  == (iconv_t)-1 ||
+        ric_gbk == (iconv_t)-1)
+            return 3;
```

Comment:
wrong indent.

Reply:
@tchaikov Hello?

Reply:
@yshui i am posting #77 to address your issue. could you help review/test it? thanks!